app/assets/stylesheets/homepage.css.scss

```diff
@@ -169,13 +166,13 @@ list-style-type: none;
 }
 
 .home-about-us {
-    text-agn: center;
+    text-align: center;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered height, text-align